distribution/lib/Standard/Base/0.0.0-dev/src/Data/Read/Many_Files_List.enso

```diff
@@ -0,0 +1,22 @@
+import project.Data.Text.Text
+import project.Data.Vector.Vector
+
+## A common interface that represents a list of files that can be read.
+
+   Various types (e.g. Vector, Column) can convert to this type to be able to be
+   used in `Data.read_many`.
+type Many_Files_List
+    ## PRIVATE
```

Comment:
Why this constructor isn't `private`?

Reply:
Because it is part of public API of Standard.Base intended for library developers. It is PRIVATE for GUI users, but library developers are supposed to implement conversions into `Many_Files_List`. See usages in `Standard.Table`.